src/app/chip_data_model.gni

```diff
@@ -248,278 +248,31 @@ template("chip_data_model") {
       defines = []
     }
 
+    # Clusters are expected to have the following format:
+    #   - A `BUILD.gn` file that defines generic targets. A dependency
+    #     will be added on this default target. An empty group will be created
+    #     if no special build rules exist
+    #   - codegen_sources.gni - required sources for code generation integration
+    #     MUST define a variable named `codegen_sources`
```

Comment:
Thanks for explaining the setup to me on Slack.  It think the right name for the variable is `codegen_dependent_sources` or `sources_depending_on_codegen`.

Or maybe `sources_depending_on_generated_app_config` to be _really_ clear.

Reply:
Updated to `codegen_dependent_sources`. The `..._on_generated_app_config` feels a bit too long (even if it is the most accurate).